server/server.js

```diff
@@ -26,34 +26,56 @@ const path = require('path');
 const randomId = require('random-id');
 const cfg = require('./config.js');
 const jwt = require('jsonwebtoken');
-const { globalConf } = require('../dashboard/public/config.js')
 const axios = require('axios');
 const bodyParser = require('body-parser');
-cfg.config('../dashboard/dist/index.html');
-cfg.L.info('server config ', cfg.dashboardConfig)
+cfg.L.info('Global config ', cfg.globalConf)
```

Comment:
Might want to remove this if we don't want to expose all the configs

Reply:
Good point. How about switching it to debug level? Sometimes it useful to easily get this when troubleshooting.